### PR TITLE
doc: Added a link to Ceph Community Calendar.

### PR DESCRIPTION
**Pull request by zdover23**
Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/start/get-involved.rst [hunk 1]
```diff
@@ -70,6 +70,8 @@ These are exciting times in the Ceph community! Get involved!
 |                      | at http://github.com. See `Ceph Source Code`_   |                                               |
 |                      | for details on cloning from github.             |                                               |
 +----------------------+-------------------------------------------------+-----------------------------------------------+
+| **Ceph Calendar**    | Learn about upcoming Ceph events.               | https://ceph.io.contribute/                   |
```
Comment (neha-ojha) [inline doc/start/get-involved.rst]:
```suggestion
| **Ceph Calendar**    | Learn about upcoming Ceph events.               | https://ceph.io/contribute/                   |
```

**Comment by neha-ojha** [comment]
@zdover23 You'll have to squash the commits and force push to make the signed-off-by check happy.

**Comment by zdover23** [comment]
I think I might have squashed in an acceptable way. Let's see if I'm right.

**Comment by neha-ojha** [comment]
> I think I might have squashed in an acceptable way. Let's see if I'm right.

Don't think it worked, what steps did you use to squash your commits?

**Comment by badone** [comment]
jenkins render docs

**Comment by neha-ojha** [comment]
> LGTM @neha-ojha what say you? :)

I think my previous code review comment still holds for this version, maybe something got lost in the rebase?

**Comment by badone** [comment]
@neha-ojha Drat, so it does. One sec.

**Comment by badone** [comment]
jenkins render docs

**Comment by badone** [comment]
@zdover23 Looks better to me mate

**Pull request merged**